content/exercise-linear-tikhonov-inversion.md

```diff
@@ -63,7 +63,7 @@ The numerical details regarding how we obtain a solution to the inverse problem
   - _Do you get the same model with different ranges of $\beta$ values?_
   - _What happens the range of $\beta$ values is too small? - try several small ranges_
   - _What happens if are only a few values of $\beta$ over a large range?_
-  - _Does the optimal value $\beta^_$ stay the same or similar when any of the changes in this investigation are made?\*
+  - _Does the optimal value {math}`\beta^*` stay the same or similar when any of the changes in this investigation are made?_
```

Comment:
This is the only content change that I didn't catch in my formatter - changing unclosed "italics" across math. Putting then in a fence protects that class of errors.